### PR TITLE
refactor: remove unused code and fix typos

### PR DESCRIPTION
**Pull request by colinlyguo**
### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [x] refactor: A code change that doesn't fix a bug, or add a feature, or improves performance

### CODE REVIEW

File: tests/integration-test/genesis.json [hunk 1]
```diff
@@ -14,9 +14,6 @@
     "londonBlock": 0,
     "archimedesBlock": 0,
     "shanghaiBlock": 0,
-    "bernoulliBlock": 2,
-    "curieBlock": 3,
-    "darwinTime": 1719558098,
```
Comment (georgehao) [inline tests/integration-test/genesis.json:19]: why delete this?

Comment (colinlyguo) [reply tests/integration-test/genesis.json:19]: to make the diff compared with `develop` branch smaller, I thought this was added by accident.

Comment (georgehao) [reply tests/integration-test/genesis.json:19]: no, intentional addition. can change it all to 0

Comment (colinlyguo) [reply tests/integration-test/genesis.json:19]: OK.

Comment (colinlyguo) [reply tests/integration-test/genesis.json:19]: added back in [4645ccf](https://github.com/scroll-tech/scroll/pull/1470/commits/4645ccffae4d62f59d3824db6d1b523a039a0263).

**Review by amoylan2 — APPROVED** [review]
LGTM

**Pull request merged**